css/styles.css

```diff
@@ -0,0 +1,43 @@
+/* below is the CSS for the header */
+
+header {
+  background-color: #4c2676;
+  color: white;
+}
+
+.padding-20 {
+  padding: 10px;
+}
+
+/* below is the CSS for the body */
+
+.marginTopBottom10 {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
```

Comment:
You could potentially shorten this to being `margin-y-10` or `my-10` to make the class shorter and still descriptive.